build/commands/lib/config.js

```diff
@@ -686,7 +686,7 @@ Config.prototype.buildArgs = function () {
     // in the future to see if this is no longer needed
     // https://github.com/brave/brave-browser/issues/29934
     args.ios_partition_alloc_enabled = false
-    args.use_partition_alloc = false
+    args.use_partition_alloc = true
```

Comment:
I'm confused, upstream now defaults to false so we are enabling? Can't we just switch to the defaults for both `ios_partition_alloc_enabled` and `use_partition_alloc` now? 

Also it seems fine with both disabled here https://source.chromium.org/chromium/chromium/src/+/main:third_party/dawn/third_party/angle/build/toolchain/ios/BUILD.gn;l=49

Reply:
They're seemingly independent, that link is for the app extension targets not the main app.

Reply:
But we can try enabling the first arg as well in a future PR

Reply:
@kylehickinson yes I know that link is for something else, but they are both disabled in that and it doesn't seem to be causing a problem? In either case `use_partition_alloc` is now the default so there's no reason to have it here in config.js

Reply:
Sorry, should have clarified, the reason its not needed to be on for the extension targets is because the extension targets dont explicitly set up for it like the main app does here in `IOSChromeMainParts`: https://source.chromium.org/chromium/chromium/src/+/main:ios/chrome/browser/web/model/chrome_main_parts.mm;l=351?q=IOSChromeMainParts::PreMainMessageLoopRun

But yes you're right, the default of `use_partition_alloc` is true so we can technically remove it from the config.js